DrivewayDentDeletion/Operators/Dockerfiles/ACE-API.Dockerfile

```diff
@@ -1,2 +1,3 @@
 FROM cp.icr.io/cp/appc/ace-server-prod@sha256:dd3c1e8d204b37775b792fc25a0bad4daba4fa35cd5aad996b29b1db63959baf
+ENV MQCERTLABL=aceclient
```

Comment:
Seem to be using 2 different ways of setting env vars. Here baking into the image, later in the CR. Any reason?

Reply:
Dan.exe wasn't working for moment. so ignoring this

Reply:
Well, obviously a .exe isn't going to work on a Mac!